x/resource/utils/media_type.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+import (
+	"github.com/gabriel-vasile/mimetype"
```

Comment:
Is it not better to use this one since it's a base package? https://pkg.go.dev/mime

The package used here can only do it based on magic numbers. The base `mime` package can also additionally do it based on file extension